rules/role_rules.yaml

```diff
@@ -0,0 +1,40 @@
+# Copyright 2019 The Forseti Security Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# rules:
+#   - role_name: "forsetiBigqueryViewer"
```

Comment:
maybe change the names of these test/example roles (here and in _test.py files) to be more generic and not have 'Forseti' in, so it isn't confusing.